src/main/java/executor/service/objectfactory/ObjectFactoryImpl.java

```diff
@@ -80,6 +80,8 @@ private <T> T createNotAutoConfigureClass(Class<T> clazz) throws InstantiationEx
                 return createStepExecutionFabrice();
             } else if (clazz.isAssignableFrom(ExecutableScenarioComposerImpl.class)) {
                 return createExecutableScenarioComposerImpl();
+            } else if (clazz.isAssignableFrom(ProxyNetworkConfig.class)) {
+                return (T) getDefaultProxy();
```

Comment:
whats Default proxy?

Reply:
As I remember, on the one of the meetings, team decide to use default proxy, but I don't remember that team decide witch it is.